package.json

```diff
@@ -20,13 +20,14 @@
     "dotenv": "^16.3.1",
     "eslint-config-next": "13.5.2",
     "eslint-import-resolver-typescript": "^3.6.1",
-    "next": "13.5.2",
+    "next": "13.5.4",
     "react": "18.2.0",
     "react-dom": "18.2.0",
-    "styled-components": "^6.0.8"
+    "styled-components": "^6.0.9"
```

Comment:
Looks like you might have ran `npm i` instead of `npm ci`, so it updated some packages.

@pragyakallanagoudar do we want to keep this change to update the dependencies? It could be easier to do it now than later (to avoid hairy debugging).

Reply:
Though also it seems to be minor enough updates that we can worry about that for later

Reply:
On second thought this is likely due to `npm audit fix --force`, since no changes happened to the `package.json` when I tried to run `npm i`. Usually `npm audit fix --force` breaks something, but maybe in this case npm just updated the packages? Could explore.